projects/adrv9026/zcu102/Makefile

```diff
@@ -0,0 +1,37 @@
+###############################################################################
+## Copyright (C) 2023 Analog Devices, Inc.
+### SPDX short identifier: BSD-1-Clause
+## Auto-generated, do not modify!
+###############################################################################
+
+PROJECT_NAME := adrv9026_zcu102
+
+M_DEPS += ../common/adrv9026_bd.tcl
+M_DEPS += ../../scripts/adi_pd.tcl
+M_DEPS += ../../common/zcu102/zcu102_system_constr.xdc
+M_DEPS += ../../common/zcu102/zcu102_system_bd.tcl
+M_DEPS += ../../common/xilinx/dacfifo_bd.tcl
+M_DEPS += ../../common/xilinx/adi_fir_filter_constr.xdc
+M_DEPS += ../../common/xilinx/adi_fir_filter_bd.tcl
```

Comment:
I don't think these are used so they should be removed

Reply:
Removed